.gitignore

```diff
@@ -105,6 +105,7 @@ py3env/
 *.nupkg
 
 .testconfig
+/*/.testconfig
```

Comment:
why the / at the start? also do we need the .testconfig above since all tests will be in workers?